qiskit/utils/quantum_instance.py

```diff
@@ -155,6 +163,7 @@ def __init__(
         # transpile
         initial_layout=None,
         pass_manager=None,
+        bound_pass_manager=None,
```

Comment:
Type hint.

Reply:
I think that would produce cyclic imports, the other `pass_manager` argument also has the type hint only in the docstring 🙂